update-Kafka-in-combination (#126)
Aiven-Open/klaw-docs: docs/HowTo/clusters.md

1. Log in to Klaw.
2. Navigate to the **Environments** tab and select **Clusters**.
3. On the **Clusters** screen, select **Add Cluster**.

> 4. On the **Add Apache Kafka Cluster** screen, provide the following information:

Reviewer: ```suggestion
4. <!-- vale off -->On the **Add Kafka Cluster** screen, provide the following information:<!-- vale on -->
```

   - **Cluster type**: Choose a cluster type from the dropdown menu (Apache Kafka, Schema Registry, or Apache Kafka Connect). Based
     on the cluster type, you need to include additional information.
   - **Cluster name**: Enter a name for the cluster, for example, `Dev_cluster`.

>    - **Apache Kafka flavor**: Select a Apache Kafka flavor from the dropdown list.

Reviewer: ```suggestion
  <!-- vale off -->
   - **Kafka flavor**: Select a Apache Kafka flavor from the dropdown list.
   <!-- vale on -->
```

     - Provide the additional information required based on the **cluster type**:

>        - If you are adding a **Apache Kafka cluster**, enter the following:

Reviewer: ```suggestion
       <!-- vale off -->
       - If you are adding a **Kafka cluster**, enter the following:
       <!-- vale on -->
```

         - **Protocol**: Select either plaintext or SSL.
         - **RestAPI servers**: Enter the URL of the REST API servers, for example, `https://server:12695`.
         - **Bootstrap servers**: Enter the comma-separated list of bootstrap servers, for
           example, `server1:9092,server2:9092,server3:9092`.
       - If you are adding a **Schema Registry cluster**, enter the following:
         - **Protocol**: Select either plaintext or SSL.
         - **Bootstrap servers**: Enter the comma-separated list of bootstrap servers.

>        - If you are adding a **Apache Kafka Connect cluster**, enter the following:

Reviewer: ```suggestion
       <!-- vale off -->
       - If you are adding a **Kafka Connect cluster**, enter the following:
       <!-- vale on -->
```

         - **Protocol**: Select either plaintext or SSL.

>          - **Apache Kafka Connect servers**: Enter the URL of the Apache Kafka Connect servers, for example, server1:8083.

Reviewer: ```suggestion
         <!-- vale off -->
         - **Kafka Connect servers**: Enter the URL of the Apache Kafka Connect servers, for example, server1:8083.
         <!-- vale on -->
```
